dnsconfig.js

```diff
@@ -19,6 +19,96 @@ function loadTemplate(template) {
   };
 }
 
+/**
+ * Create A, optional AAAA, and optional HTTPS (SVCB) records for an OSM web service,
+ * with optional Cloudflare proxy support.
+ *
+ * @param {string} name                - Hostname (e.g. "www").
+ * @param {string[]} ipv4s             - Required array of IPv4 addresses.
+ * @param {string[]} [ipv6s=[]]        - Optional array of IPv6 addresses (defaults to empty).
+ * @param {boolean} [enableHttps=true] - Whether to create an HTTPS record (default: true).
+ * @param {boolean} [cloudflareProxy=false] - Whether to enable Cloudflare proxy (default: false).
+ *
+ */
+function osm_web_service(
+  name,
+  ipv4s,
+  ipv6s,
```

Comment:
Why not just take one list of names, and then do the lookups in `IPV4` and `IPV6` in the function?